src/components/ActionCard.js

```diff
@@ -22,6 +23,11 @@ const ImageWrapper = styled.div`
   min-height: 260px;
 `
 
+const Title = styled.h3`
+  line-height: 0rem;
```

Comment:
@ryancreatescopy is this needed? Looks like it's creating issues when the content needs to take up multiple lines, e.g. on https://60087128e474c647250d0c89--ethereumorg.netlify.app/en/contributing/#contributors

![Image 2021-01-20 at 10 59 17 AM](https://user-images.githubusercontent.com/8097623/105221920-9f95da80-5b0e-11eb-8ecc-bbbf0bdfde66.jpg)